js/screens/ForgotPassword/ForgotPassword.js

```diff
@@ -17,10 +18,10 @@ const ForgotPassword = () => {
           />
           <Text style={styles.logoHeader}>Speakbox</Text>
         </View>
-        <ForgotPasswordForm />
+        <ForgotPasswordForm navigation={navigation} />
       </View>
     </ImageBackground>
   )
 }
 
-export default ForgotPassword
+export default withNavigation(ForgotPassword)
```

Comment:
we do not use `withNavigation` function. Please use navigation prop that has been provided to you by the react navigation